### ISSUE TITLE
issue with microsoft visual 2017

### ISSUE THREAD

**Issue by kempol** [xyzz/acquisition#433]
does acquisition really need visual 2017? because my pc cant install it for some reason 
would be nice if there is a solution to this

**Comment by xyzz** [comment]
Yes, it does require 2017 redist, what's the exact problem you have? what's your os?

**Comment by kempol** [comment]
W7
Haven't updated my windows in a long time because windows updates wouldn't work and caused me unable to install 2017 redist

**Comment by nEVER4BoRN** [comment]
Will this be fixed? Im on Vista atm and as i see V7 gona get same problem so that dont help me.
I honessly dont intend to change OS just becouse if this. 
OS older then V7 and V7 with no patch will no longer worck with acquisition.
Well it whas good run,

**Comment by xyzz** [comment]
According to https://www.visualstudio.com/en-us/productinfo/vs2017-system-requirements-vs both windows 7 sp1 and windows vista sp2 should be supported by redist, so update your OS and it should work fine.

**Comment by Ragnar7L** [comment]
JUST DO VERSION WITHOUT 2017 REDIST PLS
